front/lib/api/assistant/actions/dust_app_run.ts

```diff
@@ -357,8 +361,12 @@ export async function* runDustApp(
   // Let's run the app now.
   const now = Date.now();
 
-  const prodCredentials = await prodAPICredentialsForOwner(owner);
-  const api = new DustAPI(prodCredentials);
+  const prodCredentials = await prodAPICredentialsForOwner(owner, {
+    useLocalInDev: true,
```

Comment:
Should this be in an env var ?

Reply:
No because we want it for the app but not the actions